### PR TITLE
Sponsors and Footers v1.0

### PR DESCRIPTION
**Pull request by vrajparesh**

### CODE REVIEW

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Footer.scss';
+
+
+function Footer() {
+    return (
+        <div>
+            <div className="hrContainer"><hr color="grey" /></div>
+            <div className="Container">
```
Comment (Monal5031) [inline src/Components/Footer/Footer.jsx:9]: Would be useful to have class names related to blocks

File: src/Components/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    height: 14rem;
+    // font-family: Gilroy;
+    font-size: 1.5rem;
+    font-weight: 300;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.13;
+    letter-spacing: normal;
+    text-align: left;
+    color: #ffffff;
```
Comment (Monal5031) [inline src/Components/Footer/Footer.scss:53]: Should be in constants

File: src/Components/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    height: 14rem;
+    //font-family: Gilroy;
+    font-size: 1rem;
+    font-weight: 300;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.15;
+    letter-spacing: normal;
+    text-align: left;
+    color: #ffffff;
```
Comment (Monal5031) [inline src/Components/Footer/Footer.scss:72]: same here

File: src/Components/Footer/Footer.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    padding-left: 10rem;
+    
+}
+
+.Linkname {
+    color: #ffffff
```
Comment (Monal5031) [inline src/Components/Footer/Footer.scss:78]: same here

File: src/Components/Footer/Footer.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    height: 14rem;
+    //font-family: Gilroy;
+    font-size: 1rem;
+    font-weight: 300;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.15;
+    letter-spacing: normal;
+    text-align: left;
+    color: #ffffff;
```
Comment (Monal5031) [inline src/Components/Footer/Footer.scss:101]: same here

File: src/Components/Footer/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Footer.scss';
+
+
+function Footer() {
+    return (
+        <div>
+            <div className="hrContainer"><hr color="grey" /></div>
```
Comment (Monal5031) [inline src/Components/Footer/Footer.jsx:8]: @rossoskull sure about camelCase class names?

**Review by Monal5031 — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
@Monal5031 ko ignore kyo kar rahe ho juniors 😈

**Comment by Monal5031** [comment]
@anshumanv Task kam ho gaye kya office ke?

**Comment by anshumanv** [comment]
I no task, I tp 😛

**Comment by Monal5031** [comment]
@PiyushPawar17 Manager ko tag kar toh yaha :joy: Free ke paise de raha voh

**Comment by piyush-pawar-17** [comment]
@Monal5031 Next mission - Get **25 kills** in **Guardian: Overpass**

PS: No manager here

**Comment by harshendrashah** [comment]
Somebody missing UX here?

**Pull request merged**